### PR TITLE
Stored surveys

### PR DESCRIPTION
**Pull request by palmour**

### CODE REVIEW

File: www/js/input-page.js [hunk 1]
```diff
@@ -674,7 +673,8 @@ var toolTip = function(toolTipLocation){
 //Calls submitArthropodsToServer if survey upload is successful
 var submitSurveyToServer = function(){
 //	navigator.notification.alert("Submitting survey");
-	$.ajax({
+		window.location = "homepage.html"; //just fake it
```
Comment (NickWu007) [inline www/js/input-page.js]: Why go straight to homepage without making the HTTP call?

File: www/js/input-page.js [hunk 2]
```diff
@@ -699,6 +699,7 @@ var submitSurveyToServer = function(){
 		}),
 		success: function(result){
 			//Upload leaf photo
+			window.location = "homepage.html";
```
Comment (NickWu007) [inline www/js/input-page.js]: Same as above.

File: www/js/login.js [hunk 3]
```diff
@@ -33,6 +33,9 @@ function onDeviceReady(){
         }, function(error){
             alert("Transaction Error: "+error.message);
         }, function() {
+            alert("successfully retrieved cached user info.");
+            //alert("successful get" +rs.rows.item(0).usrID);
+
```
Comment (NickWu007) [inline www/js/login.js]: delete this please. This has nothing to do with pending survey

File: www/js/pending.js [hunk 4]
```diff
@@ -1,8 +1,60 @@
...
 */
+var db;
+var survey_result;
+document.addEventListener("deviceready", onDeviceReady, false);
+function onDeviceReady(){
+    
+    document.addEventListener("backbutton", function(e){
+        db.close();
+        e.preventDefault();
+        window.location.assign("StartScreen.html");
```
Comment (NickWu007) [inline www/js/pending.js]: This should go back to homepage, instead of start screen.

File: www/js/pending.js [hunk 5]
```diff
@@ -1,8 +1,60 @@
...
+            list_content += new_list_item;
+        }
+
+        $(".survey_list").html(list_content);
+        numSurveys();
+    });
+}
+
 $(document).ready(function(){
+    numSurveys();
```
Comment (NickWu007) [inline www/js/pending.js:53]: No need to put an extra function to do this, just put the logic in ready function is fine.

File: www/js/start-screen.js [hunk 6]
```diff
@@ -48,18 +48,19 @@ $(document).ready(function() {
             //tx.executeSql("DROP TABLE IF EXISTS SITE");
             tx.executeSql("CREATE TABLE IF NOT EXISTS SITE (siteId, siteName, userId, circle, state)");
             tx.executeSql('SELECT count(*) AS NUM from USER_INFO',[], function(tx, rs){
+
```
Comment (NickWu007) [inline www/js/start-screen.js]: please don't add extra newlines.

File: www/pending.html [hunk 7]
```diff
@@ -73,7 +73,7 @@
                     <div class="survey_delete text-center white-text">
                         Delete this Survey
                     </div>
-                </li> 
+                </li> -->
```
Comment (NickWu007) [inline www/pending.html]: If you are not going to use this in the future, I think you can just delete it.

File: www/survey.html [hunk 8]
```diff
@@ -192,8 +192,11 @@
 		</div>
 
 		<div class="wrapper">
+
```
Comment (NickWu007) [inline www/survey.html]: Again extra newlines.

**Review by NickWu007 — CHANGES_REQUESTED** [review]

File: www/js/pending.js (hunk 5 above)
Comment (palmour) [reply www/js/pending.js:53]: I thought it would be better to have it as a separate function since I call it in lines 53 and 48.

Comment (NickWu007) [reply www/js/pending.js:53]: Oh my bad. Thought this is only called once.

**Review by NickWu007 — APPROVED** [review]

**Pull request merged**